base/BUILD

```diff
@@ -18,18 +19,20 @@ load(
     "intellij_integration_test_suite",
     "intellij_unit_test_suite",
 )
-load("@rules_kotlin//kotlin:jvm.bzl", "kt_jvm_library")
 
 kt_jvm_library(
     name = "base",
-    srcs = glob(["src/**/*.java", "src/**/*.kt"]),
-    resources = glob(["src/resources/**/*"]),
+    srcs = glob([
+        "src/**/*.java",
+        "src/**/*.kt",
+    ]),
     resource_strip_prefix = "base/src",
+    resources = glob(["src/resources/**/*"]),
     visibility = PLUGIN_PACKAGES_VISIBILITY,
     deps = [
         "//aspect:aspect_lib",
-        "//aspect:aspect_template_lib",
         "//aspect:aspect_qsync_lib",
+        "//aspect:aspect_template_lib",
         "//common/actions",
         "//common/experiments",
         "//common/settings",
```

Comment:
Please do not format, this will cause AOSP merge conflicts down the line. Add just add new dependencies at the end of the list.

Reply:
You can disable auto formatting in the Plugin's settings.